test/functional/test_framework/wallet.py

```diff
@@ -0,0 +1,62 @@
+#!/usr/bin/env python3
+# Copyright (c) 2020 The Bitcoin Core developers
+# Distributed under the MIT software license, see the accompanying
+# file COPYING or http://www.opensource.org/licenses/mit-license.php.
+"""A limited-functionality wallet, which may replace a real wallet in tests"""
+
+from test_framework.address import ADDRESS_BCRT1_P2WSH_OP_TRUE
+from test_framework.messages import (
+    COIN,
+    COutPoint,
+    CTransaction,
+    CTxIn,
+    CTxInWitness,
+    CTxOut,
+)
+from test_framework.script import (
+    CScript,
+    OP_TRUE,
+)
+from test_framework.util import (
+    assert_equal,
+    hex_str_to_bytes,
+    satoshi_round,
+)
+from decimal import Decimal
+
+
+class MiniWallet:
+    def __init__(self, test_node):
+        self._test_node = test_node
+        self._utxos = []
+        self._address = ADDRESS_BCRT1_P2WSH_OP_TRUE
```

Comment:
why is the `_address` member needed? Why not always just use the `ADDRESS_BCRT1_P2WSH_OP_TRUE` constant directly?

Reply:
If the address is changed in the future, it might be easier if there is only one occurrence of `ADDRESS_BCRT1_P2WSH_OP_TRUE`. Not sure though

Reply:
I think it's easy either way, and best not to add code until it's needed. Doesn't make a big difference though